validator/testcases/content.py

```diff
@@ -258,6 +258,23 @@ def test_packed_scripts(err, xpi_package):
             err.pop_state()
 
 
+@decorator.register_test(tier=2)
+def test_signed_xpi(err, xpi_package):
+    """Checks if XPI is signed."""
+    search = set([
+        'META-INF/manifest.mf',
+        'META-INF/zigbert.rsa',
+        'META-INF/zigbert.sf'])
```

Comment:
This line is correctly indented, and the `tier` in the decorator was changed from `1` to `2` on your advice.